lib/grinder_files.dart

```diff
@@ -154,43 +353,56 @@ File getFile(String path) {
  * given path to a platform dependent path.
  */
 Directory getDir(String path) {
-  if (Platform.pathSeparator == '/') {
+  if (_sep == '/') {
     return new Directory(path);
   } else {
-    return new Directory(path.replaceAll('/', Platform.pathSeparator));
+    return new Directory(path.replaceAll('/', _sep));
   }
 }
 
-void copyFile(File srcFile, Directory destDir, [GrinderContext context]) {
-  File destFile = joinFile(destDir, [fileName(srcFile)]);
+void copy(FileSystemEntity entity, Directory destDir, [GrinderContext context]) {
+  return _copyImpl(entity, destDir, context);
+}
 
-  if (!destFile.existsSync() ||
-      srcFile.lastModifiedSync() != destFile.lastModifiedSync()) {
+void _copyImpl(FileSystemEntity entity, Directory destDir, [GrinderContext context]) {
+  if (entity is Directory) {
     if (context != null) {
```

Comment:
I think this should go in `copy` since the usage from `Path` doesn't care about `GrinderContext`s.